src/Ucommerce.Avenue.Umbraco/Controllers/FacetsController.cs

```diff
@@ -62,7 +60,7 @@ public ActionResult Index()
 
             if (ShouldDisplayFacets(category))
             {
-                IList<Facet> facets = _searchLibrary.GetFacetsFor(category.Guid, facetsForQuerying).Facets;
+                IList<Facet> facets = CatalogLibrary.GetProducts(category.Guid, facetsForQuerying.ToFacetDictionary()).Facets;
```

Comment:
Did you find a way to return just the facets, so we don't have to throw good data (the products) in the bin?

Reply:
good point, I added it